modules/apps/object/object-test/src/testFunctional/macros/ObjectRootModel.macro

```diff
@@ -0,0 +1,32 @@
+definition {
+
+	@summary = "Bind an object or multiple objects to the root object path"
```

Comment:
I think this summary would be better for something like this: "Bind one or multiple object definitions to the root object." I don't know why you're using the word  "path".

Reply:
Done [here](https://github.com/liferay-bpm-qa/liferay-portal/pull/1202/commits/953654685c974d6fcf6eb07ba1d90e9a4f1e6aac#diff-f5e4d02c87df108d14fbfe54ab65ddc41ec9a5073871ff3879a35d78ee5f4c73R3)